server/network/aoprotocol.py

```diff
@@ -295,7 +295,9 @@ def net_cmd_rm(self, _):
         """
 
         song_list = []
-        if not self.client.area.area_manager.arup_enabled:
```

Comment:
These changes are weird, why ignore the area settings? Maybe some servers want only 1 hub and no ARUP (like in my use case for a dr rp server)

Reply:
you'd end up with an empty area list

Reply:
addresses